content/docs/hooks-rules.md

```diff
@@ -46,13 +46,14 @@ npm install eslint-plugin-react-hooks --save-dev
 }
 ```
 
-In the future, we intend to include this plugin by default into Create React App and similar toolkits.
+ഭാവിയിൽ Create React App ലും മറ്റു ടൂൾകിട്ടുകളിലും ഡീഫോൾട്ടു ആയി ഉൾപെടുത്താൻ ഞങ്ങൾ ഉദ്ദേശിക്കുന്നുണ്ട്.
```

Comment:
ടൂള്‍കിറ്റ്

Reply:
addressed all of the above..let me know if I miss any